packages/lesswrong/lib/vulcan-lib/components.tsx

```diff
@@ -68,13 +68,25 @@ const DeferredComponentsTable = {};
 type C<T=any> = React.ComponentType<T>
 type HoC<O,T> = (component: C<O>) => C<T>
 
-const addClassnames = (componentName: string) => {
+type EmailRenderContextType = {
+  isEmailRender: boolean
+}
+
+export const EmailRenderContext = React.createContext<EmailRenderContextType|null>(null);
```

Comment:
A bit of unnecessary complexity here (the context could just be a boolean, rather than an object), but not a big deal.

Reply:
Hmm, for some reason that felt weird, but yeah, seems right. Will leave it as is for now.